azure-pipelines/pre-release-pipelines.yml

```diff
@@ -0,0 +1,78 @@
+trigger:
+- master
+
+variables:
+  NodeVersion: '10.x'
+
+jobs:
+  - job: 'Build'
+
+    pool:
+      vmImage: 'Ubuntu 16.04'
+
+    steps:
+      - task: NodeTool@0
+        inputs:
+          versionSpec: '$(NodeVersion)'
+        displayName: 'Install Node.js $(NodeVersion)'
+
+      - task: Npm@1
+        displayName: 'npm install'
+        inputs:
+          verbose: false
+
+      - script: 'gulp build'
+        displayName: 'gulp build'
+
+      - script: 'gulp test'
+        displayName: 'gulp test'
+        
+      - script: 'gulp dotnet/pack'
+        displayName: 'gulp dotnet/pack'
+        
+      - task: EsrpCodeSigning@1
```

Comment:
How is this pipeline triggered? As we discussed last time, we don't auto sign the code.

Reply:
Since this pipeline is internal ，we can disabled CI and  trigger it manually

Reply:
Who has the permission to trigger this? Any approve needed?

Reply:
only project azure-sdk  members can trigger it，currently there is no approve process  in build pipeline  it has in release
 Pipeline ,but it’s safe , because
 it’s internal，and we can know who triggered the pipeline from the history log